Add CI test for 32-bit OS

This happens after pion/webrtc@2.0.23...
- v2.0.23 - OK
- v2.0.24 - fail (TestDataChannelBufferedAmount/set_before_datachannel_becomes_open fail)
- v2.0.25 - fail (TestDataChannelBufferedAmount/set_before_datachannel_becomes_open fail)

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x804a7cc]

goroutine 48 [running]:
runtime/internal/atomic.Xadd64(0x90526ec, 0x4, 0x0, 0xffffffff, 0x0)
	/root/go/current/src/runtime/internal/atomic/asm_386.s:105 +0xc
github.com/pion/datachannel.(*DataChannel).WriteDataChannel(0x90526c0, 0x903dd90, 0x4, 0x4, 0x8070b00, 0x91204d0, 0x91d60c0, 0xc0000000)
	/root/go_ws/pkg/mod/github.com/pion/datachannel@v1.4.4/datachannel.go:294 +0x7e
github.com/pion/webrtc/v2.(*DataChannel).Send(0x9118360, 0x9130d90, 0x4, 0x4, 0x9120518, 0x9120a88)
	/root/go_ws/pkg/mod/github.com/pion/webrtc/v2@v2.0.24/datachannel.go:320 +0x72
```

I believe, the atomic operation (for datachannel stats we recently added) has some alignment issue. (not 100% sure yet)

This is causing crashes on win32 and android - also with 32bit.

--- a/.travis.yml
+++ b/.travis.yml
@@ -6,10 +6,8 @@ cache:
 env:
   - GO111MODULE=on
 
-notifications:
-  email: false
-  slack:
-    secure: IYjXoe03KykZ3v4GgUwGzfWRepO5DnJdxB87lSQ2IMsF6PBFSc3CaOX3GUclHIlzTdchR+PHj1jtEZZVSkgfp9amZBCcqbJTBOPG1YA6hxOvTpgeWIttMH0cmMxSCuCa4RfkuRH2+UXbjREMJ3ENau2CTMKReyW4Jddh9dREZohVmYuqN6uuBqCndYpt3Lm1Hv+T+vqxwTDdE/q0hwGMiwgvQm7N3K397e1q1mg+o4tMGwqyUIPnEPjaSKcEuOBa8Rqyl96nn+HGZK0zvNqUOxlzeRMM0VBcxe2s+zY/SuLj4OwNl1zEmIfY6Qj70t2cmT3xJvJprB4pCwR7q78b4lfpNu6rqCJPIZG/qDFT+XSuhDCmLlCO/+Uhtu11pgjV8UMNLTKJth+7hurH7oLNb7jYk9VYsiKhs41LICyDjJNzS5yPatF5xj0HOujb6Uh/pfI+9a+IpPSeXv1gBo8H3oWa6TfRhuTUS3Jc48p/jriZmgWgbKa1HKTaY9ENvAdZFfxJdrRg3Y4SKnjZcAPw7ijRIx1oaM3rHYbOTm/dj4ggho7EgTO3k8toQ5PKohrbBG5RERqHJvC47SXDt0fEjeGnAfN7Xtj0Pq8YyaFIj7CmCCGoI//2sWkK3AmjnwIuW0hUMsL3GsED+p0lsu6FX9wysJwy2Z2mTfIX/CXmB6w=
+services:
+  - docker
 
 install:
   # Manually download and install Go 1.12 instead of using gimme.
@@ -22,20 +20,20 @@ install:
   - go version
   - go env
   - go get -u github.com/matoous/godox
+  - curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | bash -s -- -b $GOPATH/bin v1.17.1
   # Install Node 11 (required for WASM tests)
+  - wget https://raw.githubusercontent.com/creationix/nvm/v0.31.0/nvm.sh -O ~/.nvm/nvm.sh
   - source ~/.nvm/nvm.sh
   - nvm install 11
   - node --version
   - npm i -g yarn
   - yarn install
 
-before_install:
-  - wget https://raw.githubusercontent.com/creationix/nvm/v0.31.0/nvm.sh -O ~/.nvm/nvm.sh
-
-before_script:
-  - curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | bash -s -- -b $GOPATH/bin v1.17.1
-
 script:
+  - bash .github/assert-contributors.sh
+  - bash .github/lint-disallowed-functions-in-library.sh
+  - bash .github/lint-commit-message.sh
+  - bash .github/lint-filename.sh
   - godox
   - golangci-lint run --build-tags quic ./...
   - cd examples && golangci-lint run --build-tags quic ./...
@@ -44,7 +42,4 @@ script:
   - go test -tags quic -coverpkg=$(go list ./... | grep -v examples | tr '\n' ',') -coverprofile=cover.out -v -race -covermode=atomic ./...
   - GOOS=js GOARCH=wasm go test -exec="./test-wasm/go_js_wasm_exec" -v .
   - bash <(curl -s https://codecov.io/bash)
-  - bash .github/assert-contributors.sh
-  - bash .github/lint-disallowed-functions-in-library.sh
-  - bash .github/lint-commit-message.sh
-  - bash .github/lint-filename.sh
+  - docker run -e "GO111MODULE=on" -v $PWD:/go/src/github.com/pion/webrtc -w /go/src/github.com/pion/webrtc -it i386/golang:1.12-buster '/usr/local/go/bin/go' 'test' '-tags' 'quic' "-coverpkg=$(go list ./... | grep -v examples | tr '\n' ',')" '-v' './...'

--- a/go.mod
+++ b/go.mod
@@ -5,12 +5,12 @@ go 1.12
 require (
 	github.com/pion/datachannel v1.4.5
 	github.com/pion/dtls v1.5.1
-	github.com/pion/ice v0.5.11
+	github.com/pion/ice v0.5.12
 	github.com/pion/logging v0.2.2
 	github.com/pion/quic v0.1.1
 	github.com/pion/rtcp v1.2.1
 	github.com/pion/rtp v1.1.3
-	github.com/pion/sctp v1.6.8
+	github.com/pion/sctp v1.6.9
 	github.com/pion/sdp/v2 v2.3.0
 	github.com/pion/srtp v1.2.6
 	github.com/pion/transport v0.8.6

